client/packages/views/index.js

```diff
@@ -13,8 +15,56 @@ import * as PackagesActions from '../state/actions';
 import * as NoticeActions from 'state/notices/actions';
 import GlobalNotices from 'components/global-notices';
 import notices from 'notices';
+import _ from 'lodash';
+import { sprintf } from 'sprintf-js';
 
 const Packages = ( props ) => {
+	const foldableActionButton = (
+		<button className="foldable-card__action foldable-card__expand" type="button">
+			<span className="screen-reader-text">{ __( 'Expand Services' ) }</span>
+			<Gridicon icon="chevron-down" size={ 24 } />
+		</button>
+	);
+
+	const renderPredefinedPackages = () => {
+		const elements = [];
+
+		_.forEach( props.form.predefinedSchema, ( servicePackages, serviceId ) => {
+			const serviceSelected = props.form.packages.predefined[ serviceId ] || [];
+
+			_.forEach( servicePackages, ( predefGroup, groupId ) => {
+				const groupPackages = predefGroup.definitions;
+				const groupPackageIds = groupPackages.map( ( def ) => def.id );
+				const diffLen = _.difference( groupPackageIds, serviceSelected ).length;
+				const summary = 0 === diffLen ? __( 'All packages selected' ) : sprintf( __( '%d packages selected' ), groupPackageIds.length - diffLen );
```

Comment:
When only 1 package is selected, this displays "1 packages selected" - let's handle that case better ("1 package selected")